### DISCUSSION TITLE
머머머머장님 요즘 바쁘심?

### DISCUSSION THREAD

**Discussion by zilrty** [RimWorldKorea/RMK#1056] [일반]
번역 수정해야 되는것들 내가 해볼랬는데

하는 방식이 있는것 같아서 괜히 내맘대로 건드렸다가

머장이 나중에 다시 할까봐 냅뒀는데

바쁘면 내일 내가 작업해서 릴리즈 할까?

**Comment by RimWorldKorea** [comment]
시간 남으면 추가건만 처리해줘
